app/helpers/script_helper.rb

```diff
@@ -23,7 +23,13 @@ def render_javascript_pack_once_tags(*names)
         [
           javascript_assets_tag(*@scripts),
           javascript_polyfill_pack_tag,
-          javascript_include_tag(*sources, crossorigin: local_crossorigin_sources? ? true : nil),
+          *sources.map do |source|
+            javascript_include_tag(
+              source,
+              crossorigin: local_crossorigin_sources? ? true : nil,
+              integrity: AssetSources.get_integrity(source),
```

Comment:
just to check my understanding
1. we fingerprint each asset in its filename so if the content changes, we'll have new URLs
2. in a 50-50 deploy state, new boxes will look for new code
3. the CDN should keep service old code for old boxes

so this should be pretty safe in our deploy setup

Reply:
Yeah, I chatted about this a bit with @mitchellhenke and I think it should be fine. Since the manifest includes both the fingerprinted JavaScript _and_ the corresponding integrity value for that file, it should always be consistent with itself. We're not really changing anything with regards to the availability of old assets in CloudFront, so I'm assuming that would continue to work as-is. And the machines reference their own local copy of the manifest, not one in CloudFront.